var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -349,6 +351,7 @@ class Mfem(Package, CudaPackage, ROCmPackage):
         depends_on(
             f"slepc+rocm amdgpu_target={gfx}", when=f"+rocm+slepc amdgpu_target={gfx} ^petsc+rocm"
         )
+    depends_on("mumps@5.1.1:", when="+mumps")
```

Comment:
@v-dobrev I also loosened this from `depends_on("mumps@5.1.1:+mumps", when="+mumps")` since it seems like the MUMPS package already includes PORD.